pocs/camera/sbig.py

```diff
@@ -32,6 +32,11 @@ def __del__(self):
 
 # Properties
 
+    @property
+    def egain(self):
+        """Image sensor gain in e-/ADU as reported by the camera."""
+        return self._info['readout modes']['RM_1X1']['gain']
```

Comment:
Will this always be there or does it need some exception handling?

Reply:
I think it will always be there. I should be accessing it via the read only property though, i.e. `self.properties['readout modes']['RM_1x1']['gain']` .

`self._info` is the dictionary of stuff that is read from the camera driver on `.connect()`. That will always contain `'readout modes'` for SBIG cameras (they're essential for taking exposures), and those `'readout modes'` will always include `'RM_1x1'` because that's the unbinned readout mode, and finally any readout mode must have a `gain` because it's part of the readout mode C `struct`.